apps/accounts/views/upstream_login.py

```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render
+from django.conf import settings
+import urllib.request as req
+
+
+def upstream_oauth_login(request):
+    mymedicare_login_url = 'https://dev2.account.mymedicare.gov/?scope=openid%20profile&client_id=bluebutton&state=54321'
+    query = 'http://localhost:8000/mymedicare/sls-callback'
```

Comment:
This should also be configurable. I'm also confused about where this points. Does this point back to the server itself? How does that work?

Reply:
@samgensburg-gov there are a couple things going on here.....

The upstream_oauth_login is designed for standard OAuth2 and based on PSA.  We could remove this for now.  Should I just remove it?

The real logic for the MyMedicare/SLS interface is in the views within `mymedicare_cb` app.

Reply:
Re: upstream_oauth_login values
Are these being updated to reference values from the settings files?

I am working on a task to set those values from the deployment scripts.